### PR TITLE
[Bug] Fix missing `.get_named_params_without_weight_decay()` in llama

### PR DESCRIPTION
**Pull request by xrsrke**

### CODE REVIEW

File: examples/mamba/mamba.py [hunk 1]
```diff
@@ -813,6 +805,14 @@ def forward(
             label_mask=label_mask,
         )["loss"]
         return {"loss": loss}
+    
+    def get_named_params_without_weight_decay(self):
```
Comment (3outeille) [inline examples/mamba/mamba.py:809]: why is it inside masked_mean?

Comment (xrsrke) [reply examples/mamba/mamba.py:809]: ?

https://github.com/huggingface/nanotron/blob/2f9e7d8884202c7b72294aacc1d55dfe1cd43100/examples/mamba/mamba.py#L809-L815

Comment (xrsrke) [reply examples/mamba/mamba.py:809]: it's in the NanotronModel

Comment (3outeille) [reply examples/mamba/mamba.py:809]: sorry was reviewing through github (displayed was buggy). Anyway, we could have kept it in `MambaModel` no ?

**Comment by 3outeille** [comment]
lgtm

**Pull request merged**